src/chat/model_selector.rs

```diff
@@ -320,55 +320,72 @@ impl ModelSelector {
     }
 
     fn update_selected_model_info(&mut self, cx: &mut Cx, store: &Store) {
+        let is_loading = store.chats.model_loader.is_loading();
+        let loaded_file = store.chats.loaded_model.as_ref();
+
+        let file = store
+            .chats
+            .get_current_chat()
+            .and_then(|c| c.borrow().last_used_file_id.clone())
+            .and_then(|file_id| store.downloads.get_file(&file_id).cloned())
+            .or_else(|| loaded_file.cloned());
+
+        let model = file
+            .as_ref()
+            .map(|f| store.downloads.get_model_by_file_id(&f.id).cloned())
+            .flatten();
+
+        let file_name = file.as_ref().map(|f| f.name.trim()).unwrap_or("");
+        let architecture = model.as_ref().map(|m| m.architecture.trim()).unwrap_or("");
+        let params_size = model.as_ref().map(|m| m.size.trim()).unwrap_or("");
+        let file_size = file
+            .as_ref()
+            .map(|f| format_model_size(f.size.trim()).ok())
+            .flatten()
+            .unwrap_or("".into());
+
+        let is_architecture_visible = !architecture.is_empty() && !is_loading;
+        let is_params_size_visible = !params_size.is_empty() && !is_loading;
+        let is_file_size_visible = !file_size.is_empty() && !is_loading;
+
+        let caption = if is_loading {
+            format!("Loading {}", file_name)
+        } else {
+            file_name.to_string()
+        };
+
+        let is_loaded_file = match (file, loaded_file) {
+            (Some(f), Some(lf)) => f.id == lf.id,
+            (None, None) => true,
+            _ => false,
+        };
+
+        let text_enabled_color = hex_rgb_color(0x000000);
```

Comment:
I don't know if makepad gives us some helper to build vec4 colors outside of the DSL so I made this util. 
This avoids needing to clarify which color this vector represents and also is recognized by vs code extensions like [colorize](https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize).